src/js/componentBoilerplate.js

```diff
@@ -12,10 +12,10 @@ class ComponentBoilerplate {
 		if (!(rootEl instanceof HTMLElement)) {
 			rootEl = document.querySelector(rootEl);
 		}
-		if (rootEl instanceof HTMLElement && /\bo-component-boilerplate\b/.test(rootEl.getAttribute('data-o-component'))) {
+		if (rootEl instanceof HTMLElement && rootEl.matches('[data-o-component=o-component-boilerplate]')) {
```

Comment:
We have a polyfill for matches and it is in the default set of polyfills :-D
https://polyfill.io/v2/docs/features/#Element_prototype_matches